setup.cfg

```diff
@@ -56,6 +59,7 @@ dev =
     pytest>=6.2.0
     setuptools>=56.0.0
     sphinx>=4.1.0
+    tensorflow>=2.7.0
```

Comment:
I don't get the difference between `tensorflow` and `keras`. We're using `keras`, so why is the `tensorflow` package required? There is also a `keras` package on PyPI...

Reply:
`keras` is the high-level interface that provides 'layers' which make it easy to build networks. `tensorflow` is the backend that implements all the math and gradient computation stuff. So we actually need both, but `keras` comes as a part of `tensorflow`, therefore the dependency is written like that.